.gitignore

```diff
@@ -44,3 +44,7 @@ testem.log
 # System Files
 .DS_Store
 Thumbs.db
+
+# Mock data
```

Comment:
I'm not sure why you'd want to ignore mock data. If you will ignore them, your tests will fail at CI. `gitignore` files won't be added to the remote at all and if you are using them anywhere else, it will fail.

Reply:
I didn't realize that, I put that in gitignore after I already pushed it to github so tests was ok so I thought everything's ok but you're right I fixed that.

Reply:
What `.gitignore` does is that it changes how it views them. That means if you've pushed these changes already, they will be deleted with next push.